lib/vrl/stdlib/src/unnest.rs

```diff
@@ -360,6 +454,7 @@ mod tests {
                 new: type_def! { array [
                     type_def! { object {
                         "nonk" => type_def! { bytes },
+                        "norg" => type_def! { unknown },
```

Comment:
This is making the type definition _more accurate_ than it was before. But I honestly can't remember the reasoning we used to came to this conclusion when we discussed this. Do you remember @StephenWakely?